### DISCUSSION TITLE
Question: How do I open a `diff` or `patch` of a commit

### DISCUSSION THREAD

**Discussion by felipe1982** [gitkraken/vscode-gitlens#4057] [Q&A]
From the "File history view" I want to open a patch of one commit on one file. I want to send this patch to someone for review that does not have access to the git repository.

Also, how would I be able to get a patch of a range of commits (1 or more files)?

**Comment by eamodio** [comment]
You can use the _Copy Changes (Patch)_ option, which will copy a patch to the clipboard

![image](https://github.com/user-attachments/assets/85c430f5-5c39-4400-8b6e-d9474e074c41)
